src/UnigramWordPredictor.java

```diff
@@ -50,10 +51,20 @@ public UnigramWordPredictor(Tokenizer tokenizer) {
    */
   public void train(Scanner scanner) {
     List<String> trainingWords = tokenizer.tokenize(scanner);
-
+    Map<String, List<String>> neighborMap = new HashMap<>();
```

Comment:
This works, but consider just using the instance variable directly instead of creating a local variable and then assigning it to the instance variable at the end.